layouts/authors/single.html

```diff
@@ -198,16 +216,45 @@ <h3 class="title-3 xs-mb-20">{{ .Site.Data.globals.enged_cta.title }}</h3>
   </div>
 </section>
 
-<script type="text/javascript">
-  const toggleToolTip = (el) => {
-    el.classList.toggle('tooltip');
-  }
+<script defer type="text/javascript">
   
-  const insightItems = document.getElementsByClassName("insight");
+    const toggleToolTip = (el) => {
+      el.classList.toggle('tooltip');
+    }
+    
+    const insightItems = document.getElementsByClassName("insight");
+
+    for (let i = 0; i < insightItems.length; i++) {
+      insightItems[i].addEventListener("touch", () => toggleToolTip(insightItems[i]))
+    }
+  document.addEventListener("DOMContentLoaded", async function() {
+    const getPageView = (url, currentUrl) => {
+      const urlRegex = /(http:\/\/localhost:1313\/)|(https:\/\/beta.section.io\/engineering-education\/)|(https:\/\/section.io\/engineering-education\/)/gi;
+      const trimmedUrl = url.replace(urlRegex, '');
+
+      return fetch(`https:\/\/api.www.section.io/pageviews/${trimmedUrl}`)
+        .then(res => res.json())
+        .then(data => {
```

Comment:
Have you considered what happens when this fetch errors? UI Still OK?

Reply:
By default the page views icon are now hidden. Only with a successful response will we display the page views. I'd rather not show them at all if they are just showing 0 (failed response).